### PR TITLE
add validator FAQ link to landing page

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

**Comment by CarlBeek** [comment]
I'd like to keep this as an internal link if possible. Can we use the template that @Battenfield proposed in #89?

**Comment by georgeweiler** [comment]
@CarlBeek Understood. I will create an FAQ route with launchpad styling and copy the content from that FAQ over.

**Comment by georgeweiler** [comment]
updated the links to primary color using the new `withArrow` prop, and added `/faq` link.
Screen Shot 2020-07-21 at 10 07 14 AM

File: src/pages/Landing/Phases/index.tsx [hunk 1]
```diff
@@ -54,15 +55,25 @@ const phases: phase[] = [
...
     linkUrl:
       'https://runtimeverification.com/blog/end-to-end-formal-verification-of-ethereum-2-0-deposit-smart-contract/',
-    link: 'Read about formal verification →',
+    link: 'Read about formal verification',
+    external: true,
+  },
+  {
+    title: 'Validators FAQ',
+    subTitle:
+      'Check out the FAQ page for more information about the roles and responsibilities of ETH 2 network validators.',
```
Comment (CarlBeek) [inline src/pages/Landing/Phases/index.tsx:73]:
```suggestion
      'Check out the FAQ page for more information about the roles and responsibilities of eth2 network validators.',
```

**Pull request merged**